main.go

```diff
@@ -3,11 +3,35 @@ package main
 import (
 	"github.com/guilhermef/go-shortener/config"
 	"github.com/guilhermef/go-shortener/handler"
+  "os"
+  "fmt"
 	"log"
 	"net/http"
+
+  newrelic "github.com/newrelic/go-agent"
+)
+
+var (
+	app newrelic.Application
 )
 
+func mustGetEnv(key string) string {
+	if val := os.Getenv(key); "" != val {
+		return val
+	}
+	panic(fmt.Sprintf("environment variable %s unset", key))
+}
+
 func main() {
+  nrCfg := newrelic.NewConfig(mustGetEnv("NEW_RELIC_APP_NAME"), mustGetEnv("NEW_RELIC_LICENSE_KEY"))
```

Comment:
Please use the [gofmt](https://golang.org/cmd/gofmt/) command to format your code.